phpstan-baseline.neon

```diff
@@ -2142,6 +2136,12 @@ parameters:
 			count: 1
 			path: includes/admin/class-wc-rest-stripe-orders-controller.php
 
+		-
+			message: '#^Call to an undefined method WC_Gateway_Stripe\:\:update_enabled_payment_methods\(\)\.$#'
+			identifier: method.notFound
+			count: 1
+			path: includes/admin/class-wc-rest-stripe-settings-controller.php
```

Comment:
It looks like we may be missing a guard or have an old definition for the type here -- we "know" this should be the UPE gateway, but the code doesn't reflect that. Maybe something for a follow-up?